### PR TITLE
New: Switched to ES6 module syntax and jsx template added (fixes: #12)

### LINKED ISSUE #12: Refactor / use ES6 syntax
Refactor to align with the current FW standards and switch to ES6 module syntax.

### PR DESCRIPTION
**Pull request by zubairslamdien**
#12

### New
* Now uses ES6 module syntax
* Updated to use a jsx template
* Refactor & simplify

### CODE REVIEW

File: templates/submitAll.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function SubmitAll (props) {
+  const {
+    buttonText,
+    ariaLabel,
+    onSubmitAllButtonClicked,
```
Comment (kirsty-hames) [inline templates/submitAll.jsx]: Remove comma

File: templates/submitAll.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  } = props;
+
+  return (
+    <div className={"btn__container"}>
```
Comment (kirsty-hames) [inline templates/submitAll.jsx]: Remove the curly braces for single classes:
`<div className="btn__container">`

File: js/SubmitAllView.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  className() {
+    return 'submit-all';
+  }
+
+  initialize() {
+    this.onSubmitAllButtonClicked = this.onSubmitAllButtonClicked.bind(this);
+
+    this.model.get('_articleView').$el.addClass('no-submit-buttons');
+
+    this.listenTo(Adapt, {
```
Comment (kirsty-hames) [inline js/SubmitAllView.js:14]: Indentation needs reducing to just 2 spaces.

**Review by kirsty-hames — CHANGES_REQUESTED** [review]
This is working as expected thanks @zubairslamdien. Other than the comments raised there's some trailing spaces that can be trimmed.

**Comment by oliverfoster** [comment]
Would you mind apply the suggestions please? @zubairslamdien

**Comment by oliverfoster** [comment]
Ready for review

**Comment by zubairslamdien** [comment]
Checked out and seems good to me. @oliverfoster

**Pull request merged**